project/Dependencies.scala

```diff
@@ -29,6 +29,7 @@ object Dependencies {
     "xml-resolver" % "xml-resolver" % "1.2",
     "commons-io" % "commons-io" % "2.6",
     "jline" % "jline" % "2.14.6",
+    "junit" % "junit" % "4.12", // needed for TDML runner use of AssertionViolationException
```

Comment:
I'd avoid setting junit as a core dependency to all projects. It's really only needed as a compile dependency for the TDML project--the other projects don't need it. Maybe the best way to do this is to create a new junit dependency val, and add just that to the tdml project, e.g.:

```scala
// Dependencies.scala
lazy val junit = "junit" % "junit" % "4.12"

lazy val test = Seq(
  junit % "it,test",
  "com.novocode" % "junit-interface" % "0.11" % "it,test",
  "org.scalacheck" %% "scalacheck" % "1.14.0" % "it,test"
)

// build.sbt
lazy val tdml = Project("daffodil-tdml", ...)
                ...
                .settings(libraryDependencies += Dependencies.junit)
```

Reply:
Ah, that's the right way to do it.  Thanks.